backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/docvalues/impl/AbstractSortedDocValues.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene.lowlevel.docvalues.impl;
+
+import java.io.IOException;
+import org.apache.lucene.index.SortedDocValues;
+
+/**
+ * Base implementation that throws an {@link IOException} for the
+ * {@link DocIdSetIterator} APIs. This impl is safe to use for sorting and
+ * aggregations, which only use {@link #advanceExact(int)} and
+ * {@link #ordValue()}.
```

Comment:
Same here, please add a few lines to give credit.

Reply:
It was added earlier.